scripts/in_container/run_prepare_airflow_packages.sh

```diff
@@ -34,7 +34,7 @@ function prepare_airflow_packages() {
     rm -rf -- *egg-info*
     rm -rf -- build
 
-    pip install --upgrade "pip==${AIRFLOW_PIP_VERSION}" "wheel==${WHEEL_VERSION}"
+    pip install --disable-pip-version-check "pip==${AIRFLOW_PIP_VERSION}" "wheel==${WHEEL_VERSION}"
```

Comment:
It may be a good idea to set `PIP_DISABLE_PIP_VERSION_CHECK` globally to disable the warning, since `AIRFLOW_PIP_VERSION` hardcodes the pip version, and the warning does not make too much sense when you do that.

Reply:
Good point. Will add it.

Reply:
Ah... I know why it's good this way. I just recalled why.

I actually want to have the warning if new version of PIP is released. The problem that i wanted to silence here was that pip which comes preinstalled in the image will be almost always an older version and it will also print the warnign when I actually want to upgrade PIP.

So:

* From the python image I got PIP 20.0.0 (for example)
* When PIP 21.3.4 (which we chose as the supported one) is the newest one, then we run `pip install --disable-pip-version-check pip==21.3.4` - I do not want to see the warning here that PIP 20.0.0 is outdated (I know it - that's why I want to upgrade to 21.3.4
* But any following `pip install` (of airflow and deps) **should** generate warning in case PIP 21.3.5 is installed. I want to see it, because this is a way for me or anyone else who notices, that we should attempt to upgrade PIP

So actually the "--disable-pip-version-check" flag only at the "pip upgrade" step makes perfect sense if I do not want to see any warning in case I am using latest released PIP version.

Reply:
This is basically chicken-egg problem :)